### ISSUE TITLE
Question about output of s3_rl_collect

### ISSUE THREAD

**Issue by panlexin** [yifan-you-37/omnihang#1]
Dear Yifan, 

I am very interested in this project and tried to replicate your code by running the file 's3_rl_collect.py' and got a .json file as output. I am not sure about the meaning of each elements in the output. I want to get the final relative pose between hanger and supporting item and success rate. I think that "succ" means whether the hanging of this pair objects is successful, "final_pred_transl" means the translation between two objects and "final_pred_aa" is the orientation between two objects (as marked in the image below)? 

Thank you very much! 
![0B836C20-DD6A-4EFA-A6C9-62A346894E99](https://user-images.githubusercontent.com/83538682/198536137-dbb72e5e-3343-4f73-b559-ba414a116ad3.jpeg)

**Comment by yifan-you-37** [comment]
Yes, final_pred_transl means translation, final_pred_aa means the orientation (in the axis-angle representation).
